packages/bundler-plugin-core/src/types.ts

```diff
@@ -214,17 +214,32 @@ export type Options = Omit<IncludeEntry, "paths"> & {
   uploadSourceMaps?: boolean;
 
   /**
-   * These options are considered experimental and subject to change.
-   *
-   * _experiments.injectBuildInformation:
-   * If set to true, the plugin will inject an additional `SENTRY_BUILD_INFO` variable.
-   * This contains information about the build, e.g. dependencies, node version and other useful data.
-   *
-   * Defaults to `false`.
-   * @hidden
+   * Options that are considered experimental and subject to change.
    */
   _experiments?: {
+    /**
+     * If set to true, the plugin will inject an additional `SENTRY_BUILD_INFO` variable.
+     * This contains information about the build, e.g. dependencies, node version and other useful data.
+     *
+     * Defaults to `false`.
+     */
     injectBuildInformation?: boolean;
+
+    /**
+     * Configuration for debug ID upload.
+     *
+     * Note: Currently only functional for Vite and Rollup.
+     */
+    debugIdUpload?: {
```

Comment:
m: Can we add the default values for include/ignore?

Reply:
I am not sure if a default for include makes sense, since you either define paths or you don't. Ignore makes sense though, so I added it: a7cf4e35dbaefc20a93aedd0c2f99063eee415bc